src/Evolution/Systems/GrMhd/ValenciaDivClean/LICENSE_1_0.txt

```diff
@@ -0,0 +1,23 @@
+Boost Software License - Version 1.0 - August 17th, 2003
```

Comment:
This file should probably be in the TOMS748 directory rather than in GrMHD?

Reply:
I don't know. @wthrowe  what do you think?

Reply:
I think so.  Why wouldn't RootFinding be correct?